sdks/v4-sdk/src/utils/v4Planner.ts

```diff
@@ -91,16 +93,20 @@ const ABI_DEFINITION: { [key in Actions]: string[] } = {
   [Actions.SWAP_EXACT_OUT]: [SWAP_EXACT_OUT_STRUCT],
 
   // Payments commands
-  [Actions.SETTLE]: ['address', 'uint256', 'bool'],
+  [Actions.SETTLE]: ['address', 'uint256', 'bool'], // currency, amount, payerIsUser
   [Actions.SETTLE_ALL]: ['address', 'uint256'],
-  [Actions.TAKE]: ['address', 'address', 'uint256'],
+  [Actions.TAKE]: ['address', 'address', 'uint256'], // currency, receiver, amount
   [Actions.TAKE_ALL]: ['address', 'uint256'],
   [Actions.TAKE_PORTION]: ['address', 'address', 'uint256'],
   [Actions.SETTLE_TAKE_PAIR]: ['address', 'address'],
   [Actions.CLOSE_CURRENCY]: ['address'],
   [Actions.SWEEP]: ['address', 'address'],
 }
 
+const FULL_DELTA_AMOUNT = 0
+const MSG_SENDER = '0x0000000000000000000000000000000000000001'
+const ADDRESS_THIS = '0x0000000000000000000000000000000000000002'
+
```

Comment:
ooh nice I made an ActionsConstant utils file, I can put these in there later when my PR gets merged

Reply:
yah feel free to move around. this file is def pretty cray